test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -813,13 +813,18 @@ TEST_P(ExtAuthzGrpcIntegrationTest, GoogleAsyncClientCreation) {
   initializeConfig();
   setDownstreamProtocol(Http::CodecClient::Type::HTTP2);
   HttpIntegrationTest::initialize();
-  initiateClientConnection(4, Headers{}, Headers{});
 
-  waitForExtAuthzRequest(expectedCheckRequest(Http::CodecClient::Type::HTTP2));
+  int expected_grpc_client_number;
```

Comment:
please init this. Also, WDYT of naming this `expected_grpc_client_creation_count` ? The wording now makes me feel like it's an index, but I suspect from how it's set below that it is a count of create calls.

Reply:
Done